src/main/features/dashboard/paths.ts

```diff
@@ -1,18 +1,7 @@
 import { RoutablePath } from 'common/router/routablePath'
 
-import { Paths as ClaimPaths } from 'claim/paths'
-import { Paths as ResponsePaths } from 'response/paths'
-import { Paths as OfferPaths } from 'offer/paths'
-
 export class Paths {
   static readonly dashboardPage = new RoutablePath('/dashboard/index')
   static readonly claimantPage = new RoutablePath('/dashboard/:externalId/claimant')
   static readonly defendantPage = new RoutablePath('/dashboard/:externalId/defendant')
-  static readonly claimStartPage = ClaimPaths.startPage
```

Comment:
I decided to remove these aliases since we aim to use global variables for paths in NJK files.